netbox/templates/rest_framework/api.html

```diff
@@ -2,6 +2,33 @@
 {% load static %}
 {% load i18n %}
 
+{% block bootstrap_theme %}
+  <link rel="stylesheet" type="text/css" href="{% static "rest_framework/css/bootstrap.min.css" %}"/>
+  <style>
```

Comment:
We shouldn't have inline CSS anywhere. Any custom styles should be defined as SASS under `project-static/styles/` along with the native stylesheets. We can build and package a custom CSS just for the REST web UI.